src/deploy/functions/release/fabricator.ts

```diff
@@ -316,6 +316,9 @@ export class Fabricator {
       await this.executor
         .run(async () => {
           try {
+            if ((await eventarc.getChannel(channel)) !== undefined) {
+              return;
+            }
```

Comment:
In L330, We explicitly check for error code 409 to see if the channel already exists. Are we saying 409 is the wrong code?

Reply:
createChannel will sometimes return 400 if the trigger exists and has triggers (which is common if you already functions listening to events on that channel), and 409 if exists and no triggers...

Reply:
:(. Can we add comment on why we can't rely on 409? Future me might accidentally remove this condition thinking that 409 check is sufficient.

Reply:
added a comment and rebased on master